code/modules/food_and_drinks/recipes/tablecraft/recipes_moth.dm

```diff
@@ -484,3 +484,15 @@
 	)
 	result = /obj/item/food/soup/red_porridge
 	subcategory = CAT_MOTH
+
+/datum/crafting_recipe/food/chili_sin_carne
```

Comment:
make this modular please.

Reply:
How would I do this?
(This is my first time doing anything with byond coding and github so im not really sure what you mean)

Reply:
put the recipe datum in `modular_skyrat/modules/customization/datums/components/crafting/recipes.dm`

Reply:
So the way I made it now right?